### PR TITLE
UX screen review fixes for Admin Console Teams/Channels

### PR DESCRIPTION
**Pull request by reflog**
#### Summary
Fixes after the UX screen review for Admin Console Teams/Channels

#### Related Pull Requests
https://github.com/mattermost/mattermost-webapp/pull/3017

### CODE REVIEW

File: components/admin_console/team_channel_settings/channel/details/channel_details.jsx [hunk 1]
```diff
@@ -56,10 +57,14 @@ export default class ChannelDetails extends React.Component {
         };
     }
 
-    componentDidUpdate(prevProps) { // TODO: find out how to do this without the lifecycle
-        if (prevProps.totalGroups !== this.props.totalGroups) {
+    componentDidUpdate(prevProps) {
+        if (this.props.channel !== prevProps.channel) {
```
Comment (mkraft) [inline components/admin_console/team_channel_settings/channel/details/channel_details.jsx]: You may have already, but I would just verify that this conditional isn't true every time (bearing in mind I didn't check the source of this channel prop).

File: components/admin_console/team_channel_settings/group/group_row.jsx [hunk 2]
```diff
@@ -22,7 +22,7 @@ export default class GroupRow extends React.Component {
                 className={'group '}
             >
                 <div className='group-row'>
-                    <span className='group-name'>
+                    <span className='group-name '>
```
Comment (mkraft) [inline components/admin_console/team_channel_settings/group/group_row.jsx]: Was this mistakenly staged?

**Comment by mkraft** [comment]
Oops I just noticed that this isn't in "Dev Review" state so I may have reviewed it too soon.

File: components/admin_console/team_channel_settings/group/group_row.jsx (hunk 2 above)
Comment (reflog) [reply components/admin_console/team_channel_settings/group/group_row.jsx]: 🤦‍♂

**Comment by mkraft** [comment]
@reflog I force pushed an amendment to a git message.

**Pull request merged**